clang/include/clang/CConv/ConstraintVariables.h

```diff
@@ -284,7 +284,7 @@ class PointerVariableConstraint : public ConstraintVariable {
 
   // Is an itype present for this constraint? If yes,
   // what is the text of that itype?
-  bool hasItype() const { return ItypeStr.size() > 0; }
+  bool hasItype() const override { return ItypeStr.size() > 0; }
```

Comment:
What does the `override` annotation do? Does it say you know that you are overriding a virtual method? Why is it useful to have?

Reply:
The compiler will check that this actually does override a function. If I added a `const` here but not in the super class, then the super method would not be overridden potentially casing weird errors. With the override annotation, the compile would notice that there isn't a method in the super class with the same signature.